go/kbfs/libkbfs/tlf_journal_test.go

```diff
@@ -248,6 +254,7 @@ func setupTLFJournalTest(
 		nil, nil, NewMDCacheStandard(10), ver,
 		NewReporterSimple(clocktest.NewTestClockNow(), 10), uid, verifyingKey, ekg, nil,
 		mdserver, defaultDiskLimitMaxDelay + time.Second,
+		NewMockSubscriptionManagerPublisher(gomock.NewController(t)),
```

Comment:
Why doesn't this break every test that does a `putBlockData()` call?  I would have expected an `EXPECT` somewhere.

Reply:
I think it's because the `.Finish()` method on the controller is not called anywhere: https://godoc.org/github.com/golang/mock/gomock#Controller.Finish

Reply:
Actually you're right; this breaks tests because it causes those notifications to block. Would need to `EXPECT()` those to happen `AnyTimes()`